.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,14 +6,16 @@
 
 **What this PR does / why we need it**:
 
-**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
+**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
+Fixes #
 
 **Special notes for your reviewer**:
 
 **Release note**:
-<!--  Steps to write your release note:
-1. Use the release-note-* labels to set the release note state (if you have access)
-2. Enter your extended release note in the below block; leaving it blank means using the PR title as the release note. If no release note is required, just write `NONE`.
+<!--  Write your release note:
+1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required" (case insensitive, don't include quotation marks).
+2. If no release note is required, just write "NONE" (case insensitive, don't include quotation marks).
 -->
 ```release-note
+
```

Comment:
We need to provide more guidance here, albeit briefly. Try:
"Addresses issue #(link to issue, to feature description, or to any other document that describes what the PR is a part of). (Add more details about specific work of PR). (If action required:) Users upgrading to this version must (provide details about user action required)."

Reply:
Since there are already tips in `Write your release note: 1. ... 2. ...`, do we still want the guidance in the release notes block? I'm afraid there will be people who forget to remove the guidance when they create PRs. These PRs will have `release-note` label. This will leads to noisy release notes in the final CHANGELOG. @Bradamant3 WDYT?

Some other discussion here: https://github.com/kubernetes/community/pull/1221#discussion_r145815556